tv_apps/smart-system/test/marionette/lib/system.js

```diff
@@ -67,6 +67,10 @@ System.prototype = {
     return iframe;
   },
 
+  waitForStartup: function() {
+    this.client.helper.waitForElementToDisappear('#os-logo');
+  },
+
```

Comment:
This waitForStartup method is the same as Phone's apps/system/test/marionette/lib/system.js